docs/library/datadisplay/progress.md

```diff
@@ -1,31 +1,28 @@
 ---
 components:
     - rx.progress
-    # - rx.progress.ndicator
 
 Progress: |
     lambda **props: rx.progress(value=50, **props)
```

Comment:
I think we may need to set a width on this for it to show up in the interactive example at the bottom. If it's an easy fix let's add it, else we can do it in a follow up.